docs/integrations/microsoft-azure/azure-storage.md

```diff
@@ -60,6 +61,48 @@ When you configure the event hubs source or HTTP source, plan your source catego
 
 Metrics and logs in Azure Monitor support only Azure Resource Manager storage accounts. Azure Monitor doesn't support classic storage accounts. If you want to use metrics or logs on a classic storage account, you need to migrate to an Azure Resource Manager storage account. For more information, see [Migrate to Azure Resource Manager](https://learn.microsoft.com/en-us/azure/virtual-machines/migration-classic-resource-manager-overview).
 
+## Configure Field in Field Schema
+1. In the main Sumo Logic menu, select **Manage Data** > **Logs** > **Fields**.
+2. Search for the `functionname` field.
+3. If not present, create it. Learn how to create and manage fields [here](/docs/manage/fields/#manage-fields).
+
+
+## Configure Field Extraction Rules
+Create a Field Extraction Rule (FER) for Azure Storage by following the instructions [here](/docs/manage/field-extractions/create-field-extraction-rule/).
+
+```sql title="Activity Logs Location Extraction"
+Rule Name: AzureLocationExtractionFER
+Applied at: Ingest Time
+Scope (Specific Data): tenant_name=*
+```
+
+```sql title="Parse Expression"
+json "location", "properties.resourceLocation", "properties.region" as location, resourceLocation, service_region nodrop
+| replace(toLowerCase(resourceLocation), " ", "") as resourceLocation
+| if (!isBlank(resourceLocation), resourceLocation, location) as location
+| if (!isBlank(service_region), service_region, location) as location 
+| if (isBlank(location), "global", location) as location
+| fields location
+```
+
```

Comment:
These are two different FERs probably we can add bullet points to make sure customer creates both the two FERs